packages/preload/types/electron-api.d.ts

```diff
@@ -1,5 +1,5 @@
 interface ElectronApi {
-  readonly versions: Record<string, string>
+  readonly versions: Readonly<Record<string, string>> & { [Symbol.iterator](): IterableIterator<Record<string, string>> }
```

Comment:
`IterableIterator<Record<string, string>>` should change to `IterableIterator<string>`.